freebsd/meson.build

```diff
@@ -0,0 +1,12 @@
+os_sources = files(
+	'Battery.c',
+	'FreeBSDCRT.c',
+	'FreeBSDProcess.c',
+	'FreeBSDProcessList.c',
+	'Platform.c',
+)
+
+os_inc = include_directories('.')
+
+kvm = cc.find_library('kvm', required: true)
```

Comment:
Nitpick: `required: true` is the default. This is done elsewhere too, but I will mention it just once here.